modules/ROOT/pages/manual-rollbacks.adoc

```diff
@@ -0,0 +1,12 @@
+= Manual rollbacks
+
+In case of a misbehaving OS update, it is possible to temporarily or permanently switch back to the previous working OS version.
+This is a manual operation which requires human intervention and console access.
+
+Upon updates, the previous OS deployment is kept on disk so that it can be used as a fallback if something goes wrong.
+
+In order to temporarily boot into the previous OS deployment, hold down `Shift` during OS bootup and select the relevant OS entry in the bootloader menu.
```

Comment:
Is this a grub thing? I've never heard of this.

Reply:
It is. [Here's](https://fedoraproject.org/wiki/Changes/HiddenGrubMenu#How_To_Test) an example from a test.